types/math.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	mathmod "github.com/cosmos/cosmos-sdk/math"
```

Comment:
@aaronc how do I successfully add `"github.com/cosmos/cosmos-sdk/math"` to the SDK's root `go.mod` to get this to work? I presume I have to make a (beta) release first?

Reply:
You have to tag it first yes. Temporarily you can use a `replace` directive